Nodejs/Product/TestAdapter/TestDiscoverer.cs

```diff
@@ -89,6 +89,8 @@ class TestDiscoverer : ITestDiscoverer {
                         //Debug.Fail("Before Discover");
                         DiscoverTests(testItems, proj, discoverySink, logger);
                     }
+                } catch (Exception ex) {
+                    logger.SendMessage(TestMessageLevel.Error, ex.Message);
```

Comment:
I think we should probably rethrow after logging using `throw;` to preserve the existing behavior. Let me know if that causes some sort of problem.

Other than that, the change looks good. Thank you for looking into this.